docs/index.md

```diff
@@ -1,91 +1,77 @@
 # Getting Started with Mirascope
 
-Welcome to Mirascope! This guide will help you dive into building powerful LLM-powered applications with ease. We'll start with an interactive tutorial and then guide you through next steps to deepen your knowledge and understanding of Mirascope and all of its features.
+Mirascope: Build powerful LLM applications with ease.
```

Comment:
i think we should keep this as our header "Mirascope: LLM abstractions that aren't obstructions."